server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsRequest.java

```diff
@@ -319,10 +321,20 @@ public static SortBy of(String value) {
 
         private final String value;
 
+        private final String repoName;
+
         private final String snapshotName;
 
         After(StreamInput in) throws IOException {
-            this(in.readString(), in.readString());
+            this(in.readString(), in.readString(), in.readString());
+        }
+
+        public static After fromParam(String param) {
```

Comment:
nit: perhaps call this `fromQueryParam` to signal it is the opposite of `toQueryParam`?